modules/php/Cards/GeneralStore.php

```diff
@@ -25,13 +26,14 @@ public function play($player, $args)
     parent::play($player, $args);
     $players = Players::getLivingPlayersStartingWith($player);
     Log::addAction('selection', ['players' => $players, 'src' => $this->name, 'card' => 1]);
-    Cards::createSelection(count($players));
+    $cards = Cards::createSelection(count($players));
+    $player->prepareSelection($this, $cards, $players);
     return 'selection';
```

Comment:
You can remove the return statements now that the flow is handled by the stack engine. This is dead code (if you find it somewhere, feel free to clean it as well)